pyrokinetics/local_geometry/LocalGeometry.py

```diff
@@ -175,6 +175,7 @@ def from_global_eq(
         self.dRdtheta, self.dRdr, self.dZdtheta, self.dZdr = self.get_RZ_derivatives(
             self.theta
         )
+        self.jacob = self.R * (self.dRdr * self.dZdtheta - self.dZdr * self.dRdtheta)
```

Comment:
Fine for now but we will eventually be getting all the metric terms in so will update this when that get calculated

Reply:
Yes, I agree.